src/app/features/issue/providers/github/github-common-interfaces.service.ts

```diff
@@ -174,4 +181,12 @@ export class GithubCommonInterfacesService implements IssueServiceInterface {
   private _isIssueDone(issue: GithubIssueReduced): boolean {
     return issue.state === 'closed';
   }
+
+  private _getGithubLabelsAsTags(labels: GithubLabel[]): string[] {
```

Comment:
I think, I'd take a different approach, since there is already a mechanism for adding data to thew new task in `getAddTaskData`. I'd add the additional data for the tags there and then use this info inside `IssueService.addTaskWithIssue` to trigger creating the tags there. This way we can use the same mechanism for other issue providers.

Reply:
I'm afraid I don't really get what you're saying here.. The way this is implemented, it does trigger through the `IssueService.addTaskWithIssue` function as the tags are imported as `getAddTaskData` function, right? And another Provider would need to support and implement such a functionality according to their API and restrictions anyway right? Could you outline a little more, where/how you'd like the logic to be settled?

Reply:
Still open (I don't know what mechanism you are referring to that is not already triggered/in the pipeline with the implemented approach)